clang-tools-extra/clang-tidy/modernize/UseIntegerSignComparisonCheck.cpp

```diff
@@ -117,18 +119,18 @@ void UseIntegerSignComparisonCheck::check(
   if (BinaryOp == nullptr)
     return;
 
-  auto OpCode = BinaryOp->getOpcode();
-  const auto *LHS = BinaryOp->getLHS()->IgnoreParenImpCasts();
-  const auto *RHS = BinaryOp->getRHS()->IgnoreParenImpCasts();
-  if (LHS == nullptr || RHS == nullptr)
+  const BinaryOperator::Opcode OpCode = BinaryOp->getOpcode();
+  const auto *Lhs = BinaryOp->getLHS()->IgnoreParenImpCasts();
```

Comment:
I am not sure whether is worth to change from LHS to Lhs
LHS is more wide used variable name in LLVM then Lhs. In clang source code, full uppercase abbreviation is also wide used.